docs/header.sticky.md

```diff
@@ -1,12 +1,35 @@
 # qgrid 
+
+## Option 1 - settings in template
```

Comment:
Also I have concern about scrollable container. In current implementation [sticky-header](https://github.com/titov-max/sticky-header) has special directive sticky-viewer that should be set to external container to track scroll event. 

So now I'm not sure how we should watch this in qGrid. Probably it require some improvement of sticky-header, search of external container. But I think, we doesn't provide additional options for this. If you have any ideas about possible implementation let me know.